pairinteraction.jl/src/CMakeLists.txt

```diff
@@ -15,7 +15,10 @@
 # You should have received a copy of the GNU Lesser General Public License
 # along with the pairinteraction library. If not, see <http://www.gnu.org/licenses/>.
 
-find_package(JuliaInterp REQUIRED)
+find_program(Julia_EXECUTABLE NAMES julia REQUIRED)
```

Comment:
Is it still possible to run `make` and `make test` without having Julia installed?  From my point of view, this would be desirable. Otherwise, contributors, who only want to add something to the C++ core, have to install Julia.